src/connectors/range/connectRange.js

```diff
@@ -275,11 +275,9 @@ export default function connectRange(renderFn, unmountFn = noop) {
       dispose({ state }) {
         unmountFn();
 
-        const nextState = state
+        return state
           .removeNumericRefinement(attribute)
```

Comment:
We've removed the call to `removeXRefinement` in other refactors.

Reply:
There's no function to remove a numeric facet, just a  single function that takes (or does not take) operator & value, so this is equivalent. Note that the disjunctive facet is only for the counts, and separate from the numeric

Reply:
Right!